packages/braid-design-system/codemod/bin.js

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env node
-require('./dist/index');
+require('./dist/index'); // eslint-disable-line import/no-unresolved
```

Comment:
What changed the necessitated adding this?

Reply:
Before the change, it was running build before lint